gui/wxpython/mapdisp/properties.py

```diff
@@ -229,6 +229,47 @@ def _onToggleCheckBox(self, event):
             self.giface.updateMap.emit(render=False)
 
 
+class ChBProjection:
+    """Checkbox to enable user defined projection"""
+
+    def __init__(self, parent, mapWindowProperties):
+        self.properties = mapWindowProperties
+        self.name = "projection"
+        self.label = _("Projection")
+        self.defaultLabel = _("Use defined projection")
+        self.widget = wx.CheckBox(parent=parent, id=wx.ID_ANY, label=self.defaultLabel)
+        self.widget.SetToolTip(
+            wx.ToolTip(
+                _(
+                    "Reproject coordinates displayed "
+                    "in the statusbar. Projection can be "
+                    "defined in GUI preferences dialog "
+                    "(tab 'Projection')"
+                )
+            )
+        )
+        self.widget.Bind(wx.EVT_CHECKBOX, self._onToggleCheckBox)
+
+    def GetValue(self, value):
+        self.widget.GetValue(value)
+
+    def GetWidget(self):
+        """Returns underlying widget.
+
+        :return: widget or None if doesn't exist
+        """
+        return self.widget
+

```

Comment:
Any reason this class is missing all the property functions and does not inherit from PropertyItem?

Reply:
I am a bit unsure about _connect and _disconnect methods. They are not used for projection class but they may be called (if we let projection inherit from PropertyItem). Not sure if it is not confusing. For me not, but not sure if it is the way how it should be correctly implemented.

Reply:
Well, I can share the solution with the inheritance I made...

Reply:
I don't see why it should behave/be implemented differently, so I changed that based on the other properties, but please test.

Reply:
You are right, it can be implemented in the same way. I was probably confused by signals.  I tested the changes you made and it works fine.